larsim/PhotonPropagation/PDFastSimPAR_module.cc

```diff
@@ -263,6 +263,7 @@ namespace phot {
       using ODP = fhicl::OptionalDelegatedParameter;
 
       fhicl::Atom<art::InputTag> SimulationLabel  { Name("SimulationLabel"),  Comment("SimEnergyDeposit label.") };
+      fhicl::Atom<bool>          DoFastComponent  { Name("DoFastComponent"),  Comment("Simulate slow scintillation light, default true"), true };
```

Comment:
Would experiment code fall over if the default value was removed?
If not, it might be worth considering its removal as its easier to catch FCL errors when default values aren't used.

Reply:
Yea, I included the default to maintain current behavior and keep the change non-breaking. I think the potential harm is small since I think DUNE are the only ones looking seriously at Xe doping, and that's the only circumstance I know of where it makes sense to turn off the fast component (which I think is why it wasn't an option previously).